login/token_context/token.go

```diff
@@ -0,0 +1,32 @@
+// Package token_context contains the code that extract token manager from the
+// context.
+package token_context
```

Comment:
I guess the same reason - cycling dependencies. token depends on account. Account depend on log. Log needs to depend on the context stuff.